GREEDY/DataManagers/DataConverter.cs

```diff
@@ -19,24 +19,54 @@ public List<Item> ReceiptToItemList(Receipt receipt)
             var receiptLinesToString = String.Join(Environment.NewLine, receipt.LinesOfText);
             List<Item> itemList = new List<Item>();
 
-            if (shop == "RIMI" || shop == "MAXIMA")
+            if (shop == "MAXIMA")
```

Comment:
It is usually best to avoid using == or != while comparing strings. Should use shop.Equals("MAXIMA") instead.